pkg/report/writer.go

```diff
@@ -78,16 +79,26 @@ type Option struct {
 
 // Write writes the result to output, format as passed in argument
 func Write(report types.Report, option Option) error {
+	output := os.Stdout
+	if option.Output != "" {
+		f, err := os.Create(option.Output)
+		if err != nil {
+			return xerrors.Errorf("failed to create a file: %w", err)
+		}
+		output = f
+		defer f.Close()
+	}
```

Comment:
what if move this logic to different function and use it for aws(https://github.com/aquasecurity/trivy/pull/4842/files#diff-62eef55c4c7ff14ad4fdbadd93c6348d55e1ba37b204b43e6e054a51dab293d6R62-R70) and k8s(https://github.com/aquasecurity/trivy/pull/4842/files#diff-3cf1939a3a0885c6af6d90b0f4d07617761f8d332db3523421f81e4ad373d123R99-R106)